src/v3/src/components/Widget/index.tsx

```diff
@@ -249,6 +249,11 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
       return createForm();
     }
 
+    // clear the resend reminder time stamp in session storage if current transaction does not have resend option
+    if (SessionStorage.getResendTimestamp() && !idxTransaction.nextStep?.canResend) {
+      SessionStorage.removeResendTimestamp();
+    }
```

Comment:
I noticed this logic is inside the formBag memo, I would probably recommend moving this to the useEffect that is dedicated only to the `idxTransaction` to avoid introducing a side effect here.